docs/sidebar.json

```diff
@@ -145,6 +145,11 @@
           "label": "Using PuTTY and WinSCP",
           "id": "connect-your-client/putty-winscp"
         },
+        {
+          "type": "doc",
+          "label": "Using Model Context Protocol (MCP)",
+          "id": "connect-your-client/mcp"
+        },
```

Comment:
Can we have a separate top-level section for MCP to increase visibility? It will be also easier to group all things MCP related under it as well.

We can keep this link here as well because it does make sense to have it under clients too, but I would have it link to the guide under main MCP section.